build.py

```diff
@@ -520,29 +522,31 @@ def repoagent_cmake_args(images, components, ra, install_dir):
     cargs.append('..')
     return cargs
 
+
 def cache_repo(cache):
     # example: "local", or "redis"
     return '{}_cache'.format(cache)
 
+
 def cache_cmake_args(images, components, cache, install_dir):
     args = []
 
     cargs = args + [
         cmake_cache_arg('CMAKE_BUILD_TYPE', None, FLAGS.build_type),
         cmake_cache_arg('CMAKE_INSTALL_PREFIX', 'PATH', install_dir),
         cmake_cache_arg('TRITON_COMMON_REPO_TAG', 'STRING',
-                            components['common']),
-        cmake_cache_arg('TRITON_CORE_REPO_TAG', 'STRING',
-                            components['core'])
+                        components['common']),
+        cmake_cache_arg('TRITON_CORE_REPO_TAG', 'STRING', components['core'])
     ]
 
     cargs.append(cmake_cache_enable('TRITON_ENABLE_GPU', FLAGS.enable_gpu))
     cargs += cmake_cache_extra_args()
     cargs.append('..')
     return cargs
 
+
 def backend_repo(be):
-    if (be == 'tensorflow1') or (be == 'tensorflow2'):
+    if be == 'tensorflow2':
```

Comment:
Can we not rename tensorflow2 -> tensorflow?

Reply:
Shouldn't this be "tensorflow" instead of "tensorflow2"?

Reply:
That was for changing "tensorflow2" to "tensorflow". Removed this check as the naming should have changed at [line 2216](https://github.com/triton-inference-server/server/pull/5589/files#diff-b70bfb027a8ef3ec4ab07a53a3b8ba83e63334ba3ce5ca6d9d773900ed452e5eR2216).